app/json_models.py

```diff
@@ -0,0 +1,24 @@
+class JSONModel():
```

Comment:
Technically this class has nothing to do with JSON yet, just dictionaries. Is it named with JSON for a particular reason?

Reply:
have renamed to `SerialisedModel` in 2969d8d4 and  db7c59fe